highcharts_core/chart.py

```diff
@@ -351,7 +340,7 @@ def to_js_literal(self,
         suffix = """});"""
         as_str = prefix + as_str + '\n' + suffix
 
-        if filename:
+        if filename and validators.path(filename):
```

Comment:
Can probably be further simplified to:
```suggestion
        if validators.path(filename, allow_empty = True):
```
which will evaluate to ``None`` if empty (and thus not trigger the condition) or throw an exception if not a valid path-like value.